addons/web/static/src/js/view_list.js

```diff
@@ -118,10 +118,11 @@ instance.web.ListView = instance.web.View.extend( /** @lends instance.web.ListVi
      */
     limit: function () {
         if (this._limit === undefined) {
-            this._limit = (this.options.limit
-                        || this.defaults.limit
-                        || (this.getParent().action || {}).limit
-                        || 80);
+            this._limit = (this.options.limit ||
+                        this.defaults.limit ||
+                        parseInt(this.fields_view.arch.attrs.limit, 10) ||
+                        (this.getParent().action || {}).limit ||
+                        80);
```

Comment:
That's a retarded jshint error. Please ignore it or at the very least unscrew the indentation, it's innocent and doesn't deserve to be mangled in such a way.